src/test/java/org/prebid/server/cache/CacheServiceTest.java

```diff
@@ -370,12 +370,28 @@ public void cacheBidsOpenrtbShouldTolerateGlobalTimeoutAlreadyExpired() {
         assertThat(result.getHttpCall()).isNull();
     }
 
+    @Test
+    public void cacheBidsOpenrtbShouldStoreWinUrl() throws JsonProcessingException {
+        // given
+        final com.iab.openrtb.response.Bid bid = givenBidOpenrtb(builder -> builder.id("bidId1").impid("impId1"));
+
+        // when
+        account = Account.builder().id("accountId").build();
```

Comment:
This declaration should be a part of `given` section.
Pls check all remaining tests.